clients/fides-js/src/lib/initialize.ts

```diff
@@ -394,17 +393,20 @@ export const initialize = async ({
        * with some additional client-side state so that it is initialized with
        * the user's current consent preferences, etc. and ready to display!
        */
-      const updatedExperience = updateExperience({
-        cookie: fides.cookie!,
-        experience: fides.experience,
-        isExperienceClientSideFetched: fetchedClientSideExperience,
-      });
-      fidesDebugger(
-        "Updated experience from saved preferences",
-        updatedExperience,
-      );
-      // eslint-disable-next-line no-param-reassign
-      fides.experience = { ...fides.experience, ...updatedExperience };
+      if (fetchedClientSideExperience) {
```

Comment:
This check happens here now, that's why I removed passing `isExperienceClientSideFetched` everywhere. It's essentially the same outcome but cleaner.